File: c/c99/1002_area_do_circulo.c
#include <stdio.h>
#include <math.h>

int main() {
    float raio, area, n = 3.14159;

    scanf("%f", &raio);
    area = n * pow(raio, 2);

    printf("A=%.4f\n", area);

    return 0;
}

File: java/javascript/1002_area_do_circulo.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const pi = 3.14159;
const raio =  parseFloat(lines[0]);
const area = pi * Math.pow(raio, 2);

console.log(`A=${area.toFixed(4)}`);